lib/devices/android/bootstrap/src/io/appium/android/bootstrap/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,109 @@
+package io.appium.android.bootstrap.utils;
```

Comment:
This is sooooo much better.  The fact these were all instance methods was killing me.